EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -368,6 +381,16 @@ public override string ToString()
                 sb.Append(((ParentIonMaxIntensityRanks != null) && (ParentIonMaxIntensityRanks.Any()) ? ParentIonMaxIntensityRanks.Min().ToString() : "-")); sb.Append("\t");
             }
 
+            if(ChildMatchedFragmentIons == null || ChildMatchedFragmentIons.Count == 0)
+            {
+                sb.Append("\t");
+            }
+            else
+            {
+                //TODO Fill me up!
```

Comment:
A more descriptive comment is needed to explain what's going on and what needs to be done here.